TSC_SUMMARY.md

```diff
@@ -0,0 +1,41 @@
+# Technical Steering Committee (TSC) Summary
+
+## 2023-11-02
+### Attendees: 
+- Igor Savchenko ([@DiscyDel](https://github.com/DicsyDel))
+- Marcin Wyszynski ([@marcinwyszynski](https://github.com/marcinwyszynski))
+- Roger Simms ([@allofthesepeople](https://github.com/allofthesepeople))
+- Roni Frantchi ([@roni-frantchi](https://github.com/roni-frantchi))
+
+### Absent
+- Yevgeniy Brikman ([@brikis98](https://github.com/brikis98))
+- Omry Hary ([@omry-hay](https://github.com/omry-hay))
+
+### Agenda
+
+#### Selecting an RFC for a registry solution for resolving providers/modules
+1. There was a **unanimous** consensus the RFC for [Homebrew-like artifact resolution registry component](https://github.com/opentofu/opentofu/issues/741) should be picked.
```

Comment:
We have lots of `1.` and we never hit a `2.` or a `3.` :(

Reply:
@Yantrio See [preview](https://github.com/opentofu/opentofu/pull/823/files?short_path=625d60c#diff-625d60c09be335d05265c7bd8da22e86b3df6e8d403af1be3b3efdbffed436f0) - markdown takes the number you start with and increments it as it renders